src/vision/room_luminance/scripts/perceived_luminance.py

```diff
@@ -110,12 +112,17 @@ def objectBlock(self, image_raw):
   # Regardless of the state of coverage this function returns
   # category of room light in general manner
   def classifyLuminance(self, lumene):
+    lumin = "(StateLink (AnchorNode \"luminance\")" + \
+                "(ListLink (NumberNode {})))\n".format(lumene)
+   
+    netcat(self.hostname, self.port, luminance + "\n")
```

Comment:
Vytas is referring to this: this is no longer a "pure ROS node", as it assumes the opencog infrastructure. The correct way to implement this is to do something similar to   https://github.com/opencog/ros-behavior-scripting/blob/master/sound/audioclass.py -- listen for the needed ROS events, and then place atoms into the atomspace.

By the way, what https://github.com/opencog/ros-behavior-scripting/blob/master/sound/audioclass.py is also not quite correct, I will open a bug for that -- see https://github.com/opencog/ros-behavior-scripting/issues/117 -- it does the right thing (puts audio volume into the atomspace) but also does some behavior there -- behcaiors should be psi-rules, instead of being hard-coded.